### ISSUE TITLE
Host memory stats are not updated

### ISSUE THREAD

**Issue by Rubueno** [apache/cloudstack#6621]
##### ISSUE TYPE
 * Improvement Request

##### COMPONENT NAME
~~~
LibvirtComputingResource
~~~

##### CLOUDSTACK VERSION
~~~
>= 4.15
~~~

I think this issue present in any version, although I've not looked further than 4.15. 

##### CONFIGURATION
N/A

##### OS / ENVIRONMENT
N/A

##### SUMMARY
Host memory stats are not updated frequently, leading to (possible) overcommit issues when live migrating VMs to a host. A manual restart of cloudstack-agent does update the stats, although this is not a good solution.

##### STEPS TO REPRODUCE
~~~
- Use any KVM hypervisor
- In my case in agent.properties: host.reserved.mem.mb=65536
- Migrate VMs to it until you reach capacity
- It is possible to exceed the host reserved memory
~~~

##### EXPECTED RESULTS
~~~
There should always be RAM available of at least what is specified in agent.properties: host.reserved.mem.mb
~~~

##### ACTUAL RESULTS
~~~
CloudStack will commit 100% of the host's available RAM to instances
~~~

**Comment by Rubueno** [comment]
I have proposed a patch in https://github.com/apache/cloudstack/pull/6622

**Comment by shwstppr** [comment]
Fixed with #6622
